Changed outdated type-function to is_<type>.

ejabach opened the pull request.

ejabach commented:
Forgot 2 of the  3 files. I'll update the Pull Request as soon as I fixed all files.

ghoneycutt commented on manifests/idmap.pp:
```diff
@@ -35,50 +35,32 @@
-    }
   }
-
+  
```
Please remove the trailing whitespace

ghoneycutt commented:
Could you please rebase your commits into one commit.

ejabach commented:
I rebased the commits and deleted the trailing whitespace within my last commit. In github it still says that the whitespace exists, but as soon as I fetch this commit it is gone. And the checks didn't start from new as well.
I'm new to git so it could be that I failed in the rebase process...

Phil-Friderici commented:
Thanks @ejabach, looks good to me.
Hope @ghoneycutt is fine with an additional commit for fixing ruby 1.8.7 here.

Phil-Friderici commented:
@ejabach: in the meantime, Puppet v4 compatibilty was also reached with this commit: https://github.com/ghoneycutt/puppet-module-nfs/commit/693f16edb2b6391d417833d5bc602e6182c9ea31

Therefore, I would recommend to close this PR.
